source/style.rst

```diff
@@ -0,0 +1,171 @@
+=======================
+PCDS Python Style Guide
+=======================
+As much as possible, PCDS code should be written with the following guidelines in
+mind.
+
+Code Style
+==========
+Our code follows `PEP-8`_ guidelines. Please look there for any code style matters. Since this is a very widely used standard, Google is also very helpful
+if you have any questions.
+
+Docstrings
+==========
+Please write docstrings for all modules, classes, and functions/methods. In
+general, our docstrings should mostly follow `PEP-257`_ with the exception
+of an allowance for 2-line docstrings.
+
+General Rules:
+--------------
+* Always use ``"""triple double quotes"""`` for docstrings.
+* End all summaries/descriptions with periods.
+* Built-in types (like str and int) should not be quoted or backticked.
+* Numeric values should be left unbackticked (e.g. 1).
+* Strings should be quoted (e.g. 'foo').
+* Use the ``:keyword:`` directive as appropriate (e.g. \:keyword:\`True\`).
```

Comment:
What does the keyword directive do, and why would it be appropriate here? I couldn't find documentation .

Reply:
It just formats it as a special word:
<img width="250" alt="Screen Shot 2020-04-23 at 2 28 40 PM" src="https://user-images.githubusercontent.com/6636512/80151358-c4a0f100-856e-11ea-8fc2-460219686998.png">

The ones on the left used ``` :keyword:`True` ```, the middle is just ``` `True` ```, and the right is ``` ``True`` ```.